torch/csrc/distributed/c10d/symm_mem/intra_node_comm.hpp

```diff
@@ -3,7 +3,7 @@
 #include <ATen/ATen.h>
 #include <c10/cuda/CUDAStream.h>
 #include <torch/csrc/distributed/c10d/Store.hpp>
-#include <torch/csrc/distributed/c10d/SymmetricMemory.hpp>
+#include <torch/csrc/distributed/c10d/symm_mem/SymmetricMemory.hpp>
```

Comment:
Can we apply clang-tidy to this entire subfolder in .lintrunner? <3 (optin clang-tidy / clang-format if the relevant headers pass)

Reply:
sure, it is still draft. I am update the PR right now.